locales/eu.json

```diff
@@ -0,0 +1,593 @@
+    "unmute": "Utzi mututzeari",
+    "view_other_followers": "Litekeena da beste instantziatako jarraitzaileak erakusgai ez egotea.",
+    "view_other_following": "Litekeena da beste instantziatan jarraitzen direnak erakusgai ez egotea."
+  },
+  "action": {
+    "apply": "Ezarri",
+    "bookmark": "Jarri laster-marka",
+    "bookmarked": "Laster-marka jarrita",
+    "boost": "Bultzatu",
+    "boost_count": "{0}",
+    "boosted": "Bultzatuta",
+    "clear_publish_failed": "Clear publish errors",
+    "clear_upload_failed": "Garbitu fitxategi-igoeraren erroreak",
+    "close": "Itxi",
+    "compose": "Idatzi",
+    "confirm": "Baieztatu",
+    "edit": "Editatu",
+    "enter_app": "Sartu aplikaziora",
+    "favourite": "Egin gogoko",
+    "favourite_count": "{0}",
+    "favourited": "Gogoko eginda",
+    "more": "Gehiago",
+    "next": "Hurrengoa",
+    "prev": "Aurrekoa",
+    "publish": "Argitaratu",
+    "reply": "Erantzun",
+    "reply_count": "{0}",
+    "reset": "Berrezarri",
+    "save": "Gorde",
+    "save_changes": "Gorde aldaketak",
+    "sign_in": "Hasi saioa",
+    "sign_in_to": "Hasi saioa {0}(e)n",
+    "switch_account": "Aldatu kontua",
+    "vote": "Eman botoa"
+  },
+  "app_desc_short": "Mastodon web-bezero arin bat",
+  "app_logo": "Elk Logo",
+  "app_name": "Elk",
+  "attachment": {
+    "edit_title": "Deskribapena",
+    "remove_label": "Kendu erantsitakoak"
+  },
+  "command": {
+    "activate": "Aktibatu",
+    "complete": "Osatu",
+    "compose_desc": "Idatzi bidalketa berria",
+    "n-people-in-the-past-n-days": "{0} pertsona azken {1} egunetan",
+    "select_lang": "Hautatu hizkuntza",
+    "sign_in_desc": "Gehitu dagoeneko existitzen den kontu bat",
+    "switch_account": "Aldatu {0}(e)ra",
+    "switch_account_desc": "Aldatu beste kontu batera",
+    "toggle_dark_mode": "Modu iluna",
+    "toggle_zen_mode": "ZEN modua"
+  },
+  "common": {
+    "end_of_list": "Zerrendaren amaiera",
+    "error": "ERROREA",
+    "fetching": "Eskuratzen…",
+    "in": "·",
+    "not_found": "404 Ez da aurkitu",
+    "offline_desc": "Lineaz kanpo zaudela dirudi. Egiaztatu konexioa."
+  },
+  "compose": {
+    "draft_title": "Zirriborroa {0}",
+    "drafts": "Zirriborroak ({v})"
+  },
+  "confirm": {
+    "block_account": {
+      "cancel": "Utzi",
+      "confirm": "Blokeatu",
+      "title": "Ziur {0} blokeatu nahi duzula？"
+    },
+    "block_domain": {
+      "cancel": "Utzi",
+      "confirm": "Blokeatu",
+      "title": "Ziur {0} blokeatu nahi duzula？"
+    },
+    "common": {
+      "cancel": "Ez",
+      "confirm": "Bai"
+    },
+    "delete_list": {
+      "cancel": "Utzi",
+      "confirm": "Ezabatu",
+      "title": "Ziur \"{0}\" zerrenda ezabatu nahi duzula？"
+    },
+    "delete_posts": {
+      "cancel": "Utzi",
+      "confirm": "Ezabatu",
+      "title": "Ziur bidalketa hau ezabatu nahi duzula？"
+    },
+    "mute_account": {
+      "cancel": "Utzi",
+      "confirm": "Mututu",
+      "title": "Ziur {0} mututu nahi duzula？"
```

Comment:
can you review the question mark here and on L130? check the question mark in line 140

Reply:
also on lines 111, 116 and 125